setup.cfg

```diff
@@ -21,3 +21,62 @@ include_package_data = True
 [options.entry_points]
 console_scripts =
     relenv = relenv.__main__:main
+
+[sdist]
+owner = root
+group = root
+
+[flake8]
```

Comment:
Can this config reside in `pyproject.toml`?

Reply:
Apparently no.
https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations
https://github.com/PyCQA/flake8/blob/2680339377d9fdb0e27fc1f2f30828a93c60d08a/.github/ISSUE_TEMPLATE/02_feature.yml